app/javascript/vue/BookShelf.vue

```diff
@@ -1,18 +1,26 @@
 <template>
   <div>
-    <book-card v-for="book in books" :key="book.id" :book="book"/>
+    <book-card v-for="book in booksLocal" :key="book.id" :book="book"/>
   </div>
 </template>
 
 <script>
   import BookCard from './BookCard'
```

Comment:
Let's be consistent with variable naming. Use `bookCard` instead of `BookCard`

Reply:
Isn't it a conversion for components to name them from a capital letter?

Reply:
Ah you are right my bad @resivalex and @isabloCh 😆